rest/api.go

```diff
@@ -37,21 +37,24 @@ const (
 
 // HTTP handler for the root ("/")
 func (h *handler) handleRoot() error {
+	// TODO: Lithium - remove couchdb welcome
+	resp := map[string]interface{}{
```

Comment:
If we're going to refactor away from building up raw bytes, should we go ahead and define a struct for this?  It might avoid the awkward cast on line 53?